core/templates/pages/about-page/about-page.directive.html

```diff
@@ -588,6 +588,10 @@ <h3 translate="I18N_ABOUT_PAGE_SPEECH_BUBBLE"></h3>
   .oppia-about-three-cols-feature p {
     text-align: center;
   }
+  .oppia-about-three-cols-feature,
+  .oppia-about-three-cols-feature h3 {
+    text-align: center;
+  }
```

Comment:
Why is this added?

Reply:
I applied the following code in oppia.css at line 4079.
```css

.oppia-page-card h3 {
  text-align: start;
}
```
This fixed h3 tags in the foundation tags but breaks the three-column-container at the bottom of the about tab. So I added this to revert that break and center the text as it should be.